### ISSUE TITLE
setRequested doesn't check group not being nil

### ISSUE THREAD

**Issue by gregfr** [jech/galene#77]
Greetings
I'm trying to write an alternate GUI for this wonderful server. I've run into a situation where the server crashes when I try to connect to a non-existing group. I guess the bug doesn't show in the regular GUI because it's tested beforehands.
However I've fixed it like this:

```go
// webclient.go line 680
func (c *webClient) setRequested(requested map[string]uint32) error {
	c.requested = requested

	if c.group == nil {
		return errors.New("requested group does not exist")
	}
	pushConns(c, c.group)

	return nil
}
```

Now the server doesn't crash when I'm feeding him bad parameters during the dev phase.

Regards

**Comment by jech** [comment]
That's a bug all right, but you should have received an error when you attempted to join the nonexistent group.

  https://github.com/jech/galene/blob/master/rtpconn/webclient.go#L1150

Can you confirm that you did in fact receive this error?

**Comment by jech** [comment]
Fixed in fef38bc.
